webpack.client.js

```diff
@@ -41,7 +42,8 @@ const clientConfig = {
 		'entity-editor': ['./entity-editor/controller.js'],
 		'unified-form':['./unified-form/controller.js'],
 		'entity-merge': ['./entity-editor/entity-merge.tsx'],
-		style: './stylesheets/style.scss'
+		style: './stylesheets/style.scss',
+		'relationship-type/create': ['./controllers/type-editor/relationship-type.tsx']
```

Comment:
Probably not the ideal name with a `/` in it. How about `relationship-type-editor` or `create-relationship-type`?